internal/test/e2e/run.go

```diff
@@ -411,6 +417,34 @@ func splitTinkerbellTests(awsSession *session.Session, testsList []string, conf
 	return runConfs, nil
 }
 
+func runAirgappedTinkerbellTests(awsSession *session.Session, testsList []string, conf ParallelRunConf, testRunnerConfig *TestInfraConfig, runConfs []instanceRunConf) ([]instanceRunConf, error) {
```

Comment:
Can we rename the method? it sounds like it launches and runs the actual tests but in fact it only appends test runconfig to the runConf list